src/module/system/damage/types.ts

```diff
@@ -55,18 +55,29 @@ interface DamageRollContext extends BaseRollContext {
 }
 
 interface DamageFormulaData {
-    base: BasicDamageData;
+    base: BaseDamageData | BaseDamageData[];
```

Comment:
Just make it an array from everywhere

Reply:
Sure. It'll blow up the lines of code but we can do that.